tests/models/test_hifigan_generator.py

```diff
@@ -0,0 +1,114 @@
+import math
+
+import pytest
+import torch
+
+from ami.models.hifigan_generator import HifiGANGenerator
+
+
+class TestHifiGANGenerator:
+    # model params
+    @pytest.mark.parametrize(
+        [
+            "in_channels",
+            "out_channels",
+            "upsample_rates",
+            "upsample_kernel_sizes",
+            "upsample_paddings",
+            "upsample_initial_channel",
+            "resblock_kernel_sizes",
+            "resblock_dilation_sizes",
+        ],
+        [
+            [
+                512,
+                1,
+                [8, 8, 2, 2],
+                [16, 16, 4, 4],
+                [
+                    4,
+                    4,
+                    1,
+                    1,
+                ],  # Parameters not present in the original implementation. Calculated with (upsample_kernel_size - upsample_rate) // 2
+                128,
+                [3, 7, 11],
+                [[1, 3, 5], [1, 3, 5], [1, 3, 5]],
+            ],  # cited from https://github.com/jik876/hifi-gan/blob/master/config_v1.json
+            [
+                512,
+                1,
```

Comment:
out_channels=2のケースが漏れている
```suggestion
                2,
```